src/index.ts

```diff
@@ -14,28 +15,38 @@ const configPath = process.argv[2] ?? "config.json";
 const secretsPath = process.argv[3] ?? "secrets.json";
 
 function deploySlashCommands(client: Client, config: Config) {
+	log("Deploying slash commands", logLevels.info);
 	const commands = client.guilds.cache.get(config.guildId)?.commands;
 
 	if (!commands) {
+		log('Could not deploy slash-commands. Can retry with "!deploy".', logLevels.warn);
 		return Promise.reject("Could not deploy slash-commands.");
 	}
 
 	return Promise.all(
-		getCommands().map(command =>
-			commands.create({
-				name: command.name,
-				description: command.description,
-				options: command.options,
-			})
-		)
+		getCommands().map(command => {
+			commands
+				.create({
+					name: command.name,
+					description: command.description,
+					options: command.options,
+				})
+				.then(slash => {
+					log(`Loaded ${slash.name} with id: ${slash.id}.`, logLevels.debug);
+				})
+				.catch(err => log(err, logLevels.error));
+		})
 	);
 }
 
 function main(state: State, secrets: Secrets) {
 	const { config, client } = state;
+	initLogger(state, config.debugMode);
```

Comment:
Originally I was thinking of having a bot State store (store/state.ts) and using `getState()` from within the Logger when needed, though this works too.

Reply:
When we implement #76, we will change it. For the time being this is the best I can come up with.